### ISSUE TITLE
Bold with bold italic in same line isn't converted correctly

### ISSUE THREAD

**Issue by Geroyuni** [Python-Markdown/markdown#1008]
`**Some** examples ***here***` is converted to:
`<p><strong>Some*<em> examples </em></strong>here***</p>`
This ends up looking like this in practice: <strong>Some*<em> examples </em></strong>here***

I'd expect it to be converted to something like:
`<p><strong>Some</strong> examples <strong><em>here</em></strong></p>`

![image](https://user-images.githubusercontent.com/6061850/88846195-a83ed100-d1bb-11ea-83fe-8b77d716b4bb.png)

```
C:\Users\Geroyuni>pip show markdown
Name: Markdown
Version: 3.2.2
```

**Comment by waylan** [comment]
This appears to be a duplicate of #979 which was fixed in #984 and will be included in the next release.

**Comment by Geroyuni** [comment]
I see, sorry about that if it is the case, I failed to find that issue
